index.src.html

```diff
@@ -2584,6 +2602,24 @@
         {{TypeError}}.
     2. Copy the {{EncodedVideoChunk/[[internal data]]}} into |destination|.
 
+### Serialization ###{#encodedvideochunk-serialization}
+: The {{EncodedVideoChunk}} [=serialization steps=] (with |value|, |serialized|,
+    and |forStorage|) are:
+:: 1. If |forStorage| is `true`, throw a {{TypeError}}.
+    2. For each {{EncodedVideoChunk}} internal slot in |value|, assign the value
```

Comment:
Is this sensitive to the ordering of the slots? Should the algorithms (de-)serialize each slot in a defined order?

Reply:
No I can't see how order would matter for either of these.